StringTemplate.podspec

```diff
@@ -15,4 +15,8 @@ Pod::Spec.new do |spec|
   spec.osx.deployment_target = '10.9'
   spec.watchos.deployment_target = '2.0'
   spec.tvos.deployment_target = '9.0'
+
+  spec.pod_target_xcconfig = {
+    'APPLICATION_EXTENSION_API_ONLY' => 'NO',
```

Comment:
Oh wait should this be `YES`?

Reply:
Maybe this is saying it's *only* allowable in extensions and `NO` is right? @lickel, do you know?

Reply:
Hah! bad copy paste. Fixed

Reply:
Looking at [this PR](https://github.com/squareup/cash-textfield-ios/pull/69/files), `YES` does seem correct...